src/App.js

```diff
@@ -119,9 +117,9 @@ class App extends Component {
                                 <AuthorizedRoute exact path="/contact/" component={Contact} />
                                 <AuthorizedRoute path="/contact/:uid/:action" component={Contact} />
                                 <AuthorizedRoute exact path="/reports" component={Reports} />
+                                <AuthorizedRoute exact path="/intake" component={Intake} />
```

Comment:
Let's have a chat about this today. I might be missing something, but I think that Intake is only applicable from the Contact context, therefore it might make sense to keep within the Contact directory. Additionally, I think that the route should also maintain the /contact/:uid/intake in order to pass the uid to the intake form. However, maybe there's something I missed.